src/loop/typer.ml

```diff
@@ -836,6 +836,15 @@ let non_linear_expression =
     ~hints:[text_hint]
     ~name:"Non linear expression in linear arithmetic logic" ()
 
+let bitvector_app_expected_nat =
+  Report.Error.mk ~code ~mnemonic:"bitvector-app-expected-nat"
+    ~message:(fun fmt t ->
+        Format.fprintf fmt "Expected a natural number as an argument, \
+                            but instead got the following untyped term:@ %a"
+          Dolmen_std.Term.print t)
+    ~name:"Some bitvector functions in alt-ergo require their first argument \
+           to be a literal natural number" ()
```

Comment:
That's too long  of a name ! I think you confused the name and the hint I suggested. If you want, you can take a look at the other uses of `hints` in this file, for instance : https://github.com/Gbury/dolmen/blob/master/src/loop/typer.ml#L593

Reply:
Wow! I can't believe that I pushed that, sorry! I will fix it!